<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>877</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="chartGenerated">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>791</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QPushButton" name="combineButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>190</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Combine signal</string>
    </property>
   </widget>
   <widget class="QSlider" name="frequencySlider">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>80</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="frequencyLabel">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>45</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Frequency:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="frequencyInfo">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>40</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="hzLabel">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hz</string>
    </property>
   </widget>
   <widget class="QLabel" name="signalsCombinedLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>265</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Signals combined</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>510</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="amplitudeLabel">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>130</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Amplitude:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="amplitudeInfo">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="amplitudeSlider">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>160</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QFrame" name="chartCombined">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>791</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </widget>
   <widget class="QSlider" name="wrappingFrequencySlider">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>600</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="wrappingFrequencyInfo">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>570</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::PlusMinus</enum>
    </property>
    <property name="keyboardTracking">
     <bool>true</bool>
    </property>
    <property name="showGroupSeparator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="singleStep">
     <double>0.001000000000000</double>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QFrame" name="chartFourierTransform">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>531</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4"/>
   </widget>
   <widget class="QListWidget" name="signalsList">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>290</y>
      <width>191</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="signalsHistoryLabel">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>260</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Signals history</string>
    </property>
   </widget>
   <widget class="QFrame" name="chartWrappedSignal">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>550</y>
      <width>251</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3"/>
   </widget>
   <widget class="QLabel" name="wrappingFrequencyLabel">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>570</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Frequency(ùúâ)</string>
    </property>
   </widget>
   <widget class="QLabel" name="generateSignalLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Generate signal</string>
    </property>
   </widget>
   <widget class="QLabel" name="fourierTransformLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fourier transform</string>
    </property>
   </widget>
   <widget class="QLabel" name="fourierFormulaLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>780</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;ùëìÃÇ (ùúâ)=&lt;/span&gt;&lt;span style=&quot; font-size:14pt; vertical-align:sub;&quot;&gt;t=0&lt;/span&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Œ£&lt;/span&gt;&lt;span style=&quot; font-size:14pt; vertical-align:super;&quot;&gt;T-1&lt;/span&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;ùëì(ùë°)e&lt;/span&gt;&lt;span style=&quot; font-size:14pt; vertical-align:super;&quot;&gt;‚àí2ùúãùëñùë°ùúâ&lt;/span&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;ùëëùë°&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="wrappingFormulaLabel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>780</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;e&lt;/span&gt;&lt;span style=&quot; font-size:14pt; vertical-align:super;&quot;&gt;‚àí2ùúãùëñùë°ùúâ&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1019</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
